alf/utils/data_buffer_test.py

```diff
@@ -22,16 +22,13 @@
 from absl.testing import parameterized
 
 import alf
+from alf.data_structures import Experience
```

Comment:
DataBuffer is not restricted to use Experience as its item. We should test non-Experience structure as its element.

Reply:
Now changed back to non-Experience structure based tests in data_buffer_test.